api/app/tests/utils/test_hfi_result.json

```diff
@@ -0,0 +1,3572 @@
+{
```

Comment:
Please delete, already exists in: `api/app/tests/hfi/test_hfi_result.json` on main